web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -676,6 +678,30 @@ $.extend(fixmystreet.set_up, {
             }
         });
 
+        // unhide hidden elements
+        $.each(fixmystreet.hidden_elements, function(index, element) {
+            element.show();
+        });
+        fixmystreet.hidden_elements = [];
+
+        // apply hide element rules
+        $.each(fixmystreet.bodies, function(index, body) {
+            if ( typeof hide_element_rules !== 'undefined' && hide_element_rules[body] && hide_element_rules[body][category] ) {
+                var elementIDs = hide_element_rules[body][category];
+                $.each(elementIDs, function(index, elementID) {
+                    var element = $('#' + elementID);
+                    if (!element) {
```

Comment:
`element` is a jQuery object which is truthy, `.length` is the way to check if any elements matched:

```suggestion
                    if (!element.length) {
```